package.json

```diff
@@ -20,19 +20,19 @@
         "_______ Hooks _______": "Yarn hooks.",
         "postinstall": "yarn run patch-package",
         "_______ Library Scripts #####": "Some libraries have their own build scripts.",
-        "build:libs": "lerna run build:lib",
+        "build:libs": "yarn nx run-many --target=build:lib",
         "suite:build:web": "yarn workspace @trezor/suite-web build",
         "_______ Start Scripts _______": "Here are standalone scripts for running individual applications for development.",
         "suite:dev": "yarn workspace @trezor/suite-web dev",
         "suite:dev:ios": "yarn workspace @trezor/suite-native dev:ios",
         "suite:dev:android": "yarn workspace @trezor/suite-native dev:android",
         "suite:dev:desktop": "yarn workspace @trezor/suite-desktop dev",
         "_______ Testing _______": "Testing, linting, type checking...",
-        "type-check": "yarn lerna run type-check",
+        "type-check": "yarn nx run-many --target=type-check",
         "type-check:force": "rimraf -rf -- **/libDev && yarn type-check",
-        "test:unit": "yarn lerna run test:unit",
-        "lint:js": "yarn lerna run lint:js",
-        "lint": "yarn lerna run lint:styles && yarn lint:js",
+        "test:unit": "yarn nx run-many --target=test:unit",
+        "lint:js": "yarn nx run-many --target=lint:js",
+        "lint": "yarn nx run-many --target=lint:styles && yarn lint:js",
```

Comment:
for consistency we should add standalone `lint:styles`

Reply:
https://github.com/trezor/trezor-suite/pull/6816/commits/a02869f2ab6e215b060c692bf0653ac1db663d0d